### ISSUE TITLE
NMS_REALITY_GCPROCPRODUCTTABLE Gives error

### ISSUE THREAD

**Issue by saa044** [monkeyman192/MBINCompiler#117]
I used NMS Mod Station to try and unpack NMS_REALITY_GCPROCPRODUCTTABLE.  Below is the Log.

[VERSION]: MBINCompiler v1.77.0.2
[ARGS]: "D:\AMD\nms mod\Refiner\METADATA\REALITY\TABLES\NMS_REALITY_GCPROCPRODUCTTABLE.MBIN"

If you encounter any errors, please submit a bug report and include this log file.
Please check that there isn't already a similar issue open before creating a new one.
https://github.com/monkeyman192/MBINCompiler/issues

[INFO]: Auto-Detected --input-format=MBIN
[INFO]: --input-format=MBIN --output-format=EXML

[WARN]: File not recognized. You may need to use an older version of MBINCompiler.
  [INFO]: D:\AMD\nms mod\Refiner\METADATA\REALITY\TABLES\NMS_REALITY_GCPROCPRODUCTTABLE.MBIN
  [INFO]: MBIN    version:    0.0.0.0    guid:    2062A12192E180C7    template:    cGcProceduralProductTable
  

[ERROR]: [AggregateException]: One or more errors occurred.
[MbinException]: Failed to read GcProceduralProductTable from MBIN.
[InvalidListException]: Invalid list read, magic 00000000 expected xxxxxx01 at 0x98

   at libMBIN.NMSTemplate.DeserializeList[T](BinaryReader reader, FieldInfo field, NMSAttribute settings, Int64 templateStartOffset, NMSTemplate parent)

**Comment by monkeyman192** [comment]
@saa044 do you still have this issue? If not you can close this

**Comment by hhrhhr** [comment]
the problem still exists:
```
[VERSION]: MBINCompiler v1.78.0.1
[ARGS]: "NMS_REALITY_GCPROCPRODUCTTABLE.MBIN"

--- cut ---

[WARN]: File not recognized. You may need to use an older version of MBINCompiler.
  [INFO]: b:\nms\unp\METADATA\REALITY\TABLES\NMS_REALITY_GCPROCPRODUCTTABLE.MBIN
  [INFO]: MBIN	version:	0.0.0.0	guid:	2062A12192E180C7	template:	cGcProceduralProductTable
  

[ERROR]: [AggregateException]: Произошла одна или несколько ошибок.
[MbinException]: Failed to read GcProceduralProductTable from MBIN.
[InvalidListException]: Invalid list read, magic 00000000 expected xxxxxx01 at 0x98

   в libMBIN.NMSTemplate.DeserializeList[T](BinaryReader reader, FieldInfo field, NMSAttribute settings, Int64 templateStartOffset, NMSTemplate parent)
```

**Comment by RealMiith** [comment]
same problem here:
[VERSION]: MBINCompiler v1.75.0.3
[ARGS]: "C:\Users\Miithos\Desktop\working place\NMS_REALITY_GCPROCPRODUCTTABLE.MBIN"


[INFO]: Auto-Detected --input-format=MBIN
[INFO]: --input-format=MBIN --output-format=EXML

[WARN]: File not recognized. You may need to use an older version of MBINCompiler.
  [INFO]: C:\Users\Miithos\Desktop\working place\NMS_REALITY_GCPROCPRODUCTTABLE.MBIN
  [INFO]: MBIN	version:	0.0.0.0	guid:	2062A12192E180C7	template:	cGcProceduralProductTable
  

[ERROR]: [AggregateException]: One or more errors occurred.
[MbinException]: Failed to read GcProceduralProductTable from MBIN.
[InvalidListException]: Invalid list read, magic 00000000 expected xxxxxx01

   at libMBIN.NMSTemplate.DeserializeList[T](BinaryReader reader, FieldInfo field, NMSAttribute settings, Int64 templateStartOffset, NMSTemplate parent)

**Comment by FoxStarwind** [comment]
I am having this same issue, however, I am only getting it with one file and it's from a mod I am trying to update. I am not using any programs aside from NMSPakTool to extract the MBIN file then I use MBINCompiler version 2.0.11.1 to try and decompile it into the exml file. Every mod I have updated kicks back a log file with a version of MBINCompiler that was used to create it, and after downloading that version, the rest goes smoothly.
However
The Busier Space mod on nexusmods.com gives me an MBINCompiler version of 0.0.0.0
After thinking about it and finding that the initial release of MBINCompiler was version 1.13.2, I can only assume that the mod creator used something other than MBINCompiler.
Since I only just started playing and modding NMS about a month ago, I am not aware of what might have been used prior to MBINCompiler.

[MBINCompiler.log](https://github.com/monkeyman192/MBINCompiler/files/3646253/MBINCompiler.log)
